src/snowflake/cli/api/console/console.py

```diff
@@ -78,11 +87,24 @@ def phase(self, enter_message: str, exit_message: Optional[str] = None):
             if exit_message:
                 self._print(self._format_message(exit_message, Output.PHASE))
 
+    @contextmanager
+    def indented(self) -> Iterator[Callable[[str], None]]:
+        """A context manager for indenting phase or step messages"""
+        self._extra_indent += 1
+        try:
+            yield self.message
+        finally:
+            self._extra_indent -= 1
```

Comment:
I specifically don't want to do this because indenting for more than one level should not be a common thing to do. Since the beginning the idea was the keep the API minimal and rather opinionated so that best practices are followed intuitively.